Packs/FeedCyjax/Integrations/FeedCyjax/FeedCyjax.py

```diff
@@ -0,0 +1,629 @@
+        error_msg = 'Not responding'
+
+        try:
+            indicators = list(cyjax_sdk.IndicatorOfCompromise().list(since=timedelta(minutes=5)))
+            if isinstance(indicators, list):
+                result = True
+        except Exception as e:
+            error_msg = str(e)
+            demisto.debug('Error when testing connection to Cyjax API {}'.format(error_msg))
+
+        return result, error_msg
+
+    def fetch_indicators(self, since=None, until=None, indicator_type=None, source_type=None, source_id=None,
+                         limit=None) -> list:
+        """
+        Fetch indicators from Cyjax SDK.
+
+        :type since: ``str``
+        :param since:  The start date time in ISO 8601 format.
+
+        :type until: ``str``
+        :param until:  The end date time in ISO 8601 format
+
+        :type indicator_type: ``str``
+        :param indicator_type:  The indicator type. If not specified all indicators are returned
+
+        :type source_type: ``str``
+        :param source_type:  The indicators source type. Allowed values are incidnet-report, my-report
+
+        :type source_id: ``int``
+        :param source_id:  The indicators source ID
+
+        :type limit: ``int``
+        :param limit: The indicators count limit
+
+        :return: The list of indicators
+        :rtype: list
+        """
+        try:
+            indicators = cyjax_sdk.IndicatorOfCompromise().list(since=since,
+                                                                until=until,
+                                                                type=indicator_type,
+                                                                source_type=source_type,
+                                                                source_id=source_id,
+                                                                limit=limit)
+        except Exception as e:
+            indicators = []
+            demisto.debug('Error when fetching Indicators from Cyjax SDK {}'.format(str(e)))
+
+        return indicators
+
+    def sighting(self, value: str) -> Optional[dict]:
+        """
+        Get the sighting for an indicator
+
+        :type value: ``str``
+        :param value:  The indicator value
+
+        :return: The dict with sighting metadata
+        :rtype: Optional[dict]
+        """
+        try:
+            enrichment = cyjax_sdk.IndicatorOfCompromise().enrichment(value)
+            enrichment['value'] = value
+
+            # Do not expose geoip enrichment data in sighting method
+            if 'geoip' in enrichment:
+                del enrichment['geoip']
+            if 'asn' in enrichment:
+                del enrichment['asn']
+        except Exception:
+            enrichment = None
+
+        return enrichment
+
+
+''' HELPER FUNCTIONS '''
+
+
+def arg_to_datetime(arg, arg_name=None, is_utc=True, required=False, settings=None):
+    # type: (Any, Optional[str], bool, bool, dict) -> Optional[datetime]
+
```

Comment:
Is there any good reason why not to use the same `arg_to_datetime` function that is already implemented in `CommonServerPython`?

Reply:
Fixed